nixos/tests/coder.nix

```diff
@@ -0,0 +1,38 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "coder";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ shyim ];
```

Comment:
```suggestion
    maintainers = [ shyim ghuntley ];
```

Reply:
Currently at KubeCon right now so haven't had time to properly review but thank you for digging into this. 
Can you please add myself @ghuntley as a maintainer of the package.

Reply:
Sure added with https://github.com/NixOS/nixpkgs/pull/197613/commits/d9f2bdffc952c5568bdb37cf901106184f49170c